libs/ui-watt/src/lib/components/drawer/watt-drawer.component.html

```diff
@@ -26,7 +26,13 @@
   >
     <div class="watt-drawer__grid-container">
       <header>
-        <ng-content select="watt-drawer-topbar"></ng-content>
+        <ng-content select="watt-drawer-topbar"> </ng-content>
+        <watt-button
+          class="close-btn"
+          variant="icon"
+          icon="close"
+          (click)="closed.emit()"
```

Comment:
issue: looks like using the `closed` property directly doesn't close the drawer (have tested it in DH app). Better to call `close` method instead.

```suggestion
          (click)="close()"
```